fuzzers/klee/fuzzer.py

```diff
@@ -124,20 +140,89 @@ def get_bc_files():
 
 
 def get_fuzz_target():
-    """Get the fuzz target"""
+    """Get the fuzz target name"""
+    # For non oss-projects, FUZZ_TARGET contain the target binary.
+    fuzz_target = os.getenv('FUZZ_TARGET', None)
+    if fuzz_target is not None:
+        return [fuzz_target]
+
+    print('[get_fuzz_target] FUZZ_TARGET is not defined')
+
+    # For these benchmarks, only return one file.
+    targets = {
+        'curl': 'curl_fuzzer_http',
+        'openssl': 'x509',
+        'systemd': 'fuzz-link-parser',
+        'php': 'php-fuzz-parser'
+    }
+
+    for target, fuzzname in targets.items():
+        if is_benchmark(target):
+            return [os.path.join(os.environ['OUT'], fuzzname)]
+
+    # For the reamining oss-projects, use some heuristics.
+    # We look for binaries in the OUT directory and take it as our targets.
+    # Note that we may return multiple binaries: this is necessary because
+    # sometimes multiple binaries are generated and we don't know which will
+    # be used for fuzzing (e.g., zlib benchmark).
+    # dictionary above.
     out_dir = os.environ['OUT']
-    if is_benchmark('sqlite3'):
-        return os.path.join(out_dir, 'ossfuzz')
+    files = os.listdir(out_dir)
+    fuzz_targets = []
+    for filename in files:
+        candidate_bin = os.path.join(out_dir, filename)
+        if 'fuzz' in filename and os.access(candidate_bin, os.X_OK):
+            fuzz_targets += [candidate_bin]
 
-    if is_benchmark('zlib'):
-        return os.path.join(out_dir, 'zlib_uncompress_fuzzer')
+    if len(fuzz_targets) == 0:
+        raise ValueError("Cannot find binary")
+    print("[get_fuzz_target] targets: %s" % fuzz_targets)
+    return fuzz_targets
 
-    # For non oss-projects, FUZZ_TARGET contain the target binary
-    fuzz_target = os.getenv('FUZZ_TARGET', None)
-    if fuzz_target is not None:
-        return fuzz_target
 
-    raise ValueError('Cannot determine fuzz target')
+def fix_fuzzer_lib():
```

Comment:
strange that there is so much changed for below lines. This was already in the code, right?